src/sql/workbench/contrib/welcome/page/browser/az_data_welcome_page.ts

```diff
@@ -9,69 +9,197 @@ import { localize } from 'vs/nls';
-				<div class="section recent">
-					<h2 class="caption">${escape(localize('welcomePage.recent', "Recent"))}</h2>
-					<ul class="list">
-						<!-- Filled programmatically -->
-						<li class="moreRecent"><a href="command:workbench.action.openRecent">${escape(localize('welcomePage.moreRecent', "More..."))}</a><span class="path detail if_shortcut" data-command="workbench.action.openRecent">(<span class="shortcut" data-command="workbench.action.openRecent"></span>)</span></li>
-					</ul>
-					<p class="none detail">${escape(localize('welcomePage.noRecentFolders', "No recent folders"))}</p>
+			</div>
+			<div class="ads_homepage__section content row ads_grid xl--cols-12 sm--cols-1">
+				<div class="col--lg--span-4 col--xl--start_1__span_7 col--sm--start_1__span_1 resources">
+					<h2>${escape(localize("welcomePage.resources", "Resources"))}</h2>
+					<div class="tabs">
+						<input class="input" name="tabs" type="radio" id="tab-1" checked="checked" />
+						<label class="label" for="tab-1">${escape(localize("welcomePage.history", "History"))}</label>
+						<div class="panel">
+							<div class="recent history">
+								<div class="flex flex--j_between list__header__container">
+									<h4 class="icon--document list__header">${escape(localize("welcomePage.name", "Name"))}</h4>
+									<h4>${escape(localize("welcomePage.lastOpened", "Last Opened"))}</h4>
+								</div>
+								<ul class="list">
+									<!-- Filled programmatically -->
+									<li class="moreRecent">
+										<a href="command:workbench.action.openRecent">${escape(localize("welcomePage.moreRecent", "Show more"))}</a>
+										<i class="icon--arrow_down--dark"></i>
+									</li>
+								</ul>
+								<p class="none detail">No recent folders</p>
+							</div>
+						</div>
+					</div>
+					<p class="showOnStartup"><input type="checkbox" id="showOnStartup" class="checkbox">
+						<label class="caption" for="showOnStartup">${escape(localize("welcomePage.showOnStartup", "Show welcome page on startup"))}</label>
+					</p>
 				</div>
-				<div class="section help">
-					<h2 class="caption">${escape(localize('welcomePage.help', "Help"))}</h2>
-					<ul>
-						<li><a href="https://aka.ms/get-started-azdata">${escape(localize('welcomePage.gettingStarted', "Getting started"))}</a></li>
-						<li><a href="https://aka.ms/azuredatastudio">${escape(localize('welcomePage.productDocumentation', "Documentation"))}</a></li>
-						<li><a href="https://github.com/Microsoft/azuredatastudio/issues/new/choose">${escape(localize('welcomePage.reportIssue', "Report issue or feature request"))}</a></li>
-						<li><a href="https://github.com/Microsoft/azuredatastudio">${escape(localize('welcomePage.gitHubRepository', "GitHub repository"))}</a></li>
-						<li><a href="https://aka.ms/azuredatastudio-releasenotes">${escape(localize('welcomePage.releaseNotes', "Release notes"))}</a></li>
-					</ul>
+				<div class="col--xl--start_9__span_6 col--sm--start_1__span_1">
```

Comment:
1. I couldn't find the definition for this class
2. your css seems to be generated by some tool, the naming convention is odd :)